### PR TITLE
Evernote - few enhancements (version + https + license)

### PR DESCRIPTION
**Pull request by radeksimko**
I remember there has been a discussion about `latest` VS specific versions, do I understand correctly we now prefer more specific URLs over `latest` to make the "upgrades" easier for users?

<del>Regarding license see http://en.wikipedia.org/wiki/Evernote </del>

### CODE REVIEW

File: Casks/evernote.rb [hunk 1]
```diff
@@ -3,17 +3,17 @@
...
+    version '6.0.8_451398'
+    sha256 'ff3e01a61eb176fccd604ebd409d818e7850fe1c1889a8f97f87640e77d11e69'
+    url "https://cdn1.evernote.com/mac-smd/public/Evernote_RELEASE_#{version}.dmg"
+    appcast 'https://update.evernote.com/public/ENMac/EvernoteMacUpdate.xml'
   end
 
   name 'Evernote'
   homepage 'https://evernote.com/'
-  license :gratis
+  license :freemium
```
Comment (vitorgalvao) [inline Casks/evernote.rb]: Isn’t the app itself completely free, with a possibly paid service?

https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#license-stanza-details
> a free app to access a paid service is still `:gratis`, not `:freemium`.

**Comment by vitorgalvao** [comment]
> we now prefer more specific URLs over `latest` to make the "upgrades" easier for users?

Hasn’t really been discussed at length, but eventually yes, so `upgrade` works. In casks with `appcast`, though, it makes all the sense to have them versioned, regardless of policy for the rest of them.

File: Casks/evernote.rb (hunk 1 above)
Comment (radeksimko) [inline Casks/evernote.rb]: I see what do you mean. I will omit the license change then and update the PR.

**Comment by radeksimko** [comment]
@vitorgalvao PR updated.

**Pull request merged**